### PR TITLE
Unified error handling, comments and naming.

### PR DESCRIPTION
**Pull request by rgo3**

### CODE REVIEW

File: pkg/metronome/client.go [hunk 1]
```diff
@@ -421,11 +440,14 @@ func (c *Client) Queued(jobID string) ([]Queue, error) {
 			}
 		}
 		return nil, fmt.Errorf("job '%s' does not exist", jobID)
+	case 404:
+		return nil, fmt.Errorf("job %s does not exist", jobID)
```
Comment (ArmandGrillet) [inline pkg/metronome/client.go]: Line 442: `fmt.Errorf("job '%s' does not exist", jobID)`
Here: `fmt.Errorf("job %s does not exist", jobID)`
Can we make those two messages consistent?

Comment (rgo3) [reply pkg/metronome/client.go]: Ok I'll go over all of them to make them consistent.

**Comment by rgo3** [comment]
@mesosphere-ci run all integration tests

**Comment by rgo3** [comment]
@mesosphere-ci run all integration tests

**Pull request merged**